src/streaming/stream-conf.c

```diff
@@ -1,28 +1,46 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 
-#include "stream-conf.h"
-
-struct config stream_config = APPCONFIG_INITIALIZER;
-
-bool stream_conf_send_enabled = false;
-bool stream_conf_compression_enabled = true;
-bool stream_conf_replication_enabled = true;
-
-const char *stream_conf_send_destination = NULL;
-const char *stream_conf_send_api_key = NULL;
-const char *stream_conf_send_charts_matching = "*";
-
-time_t stream_conf_replication_period = 86400;
-time_t stream_conf_replication_step = 600;
-
-const char *stream_conf_ssl_ca_path = NULL;
-const char *stream_conf_ssl_ca_file = NULL;
-
-// to have the remote netdata re-sync the charts
-// to its current clock, we send for this many
-// iterations a BEGIN line without microseconds
-// this is for the first iterations of each chart
-unsigned int stream_conf_initial_clock_resync_iterations = 60;
+#include "daemon/common.h"
+
+static struct config stream_config = APPCONFIG_INITIALIZER;
+
+struct _stream_send stream_send = {
```

Comment:
@ktsaou types/variables starting with a leading underscored are reserved for the runtime, do not use such names.

Reply:
You have to open windows.h

You will be impressed! Almost every struct and enum starts with underscore... 

I will rename them.